polaris-react/src/index.ts

```diff
@@ -52,8 +52,8 @@ export type {AvatarProps} from './components/Avatar';
 export {Backdrop} from './components/Backdrop';
 export type {BackdropProps} from './components/Backdrop';
 
-export {Badge} from './components/Badge';
-export type {BadgeProps} from './components/Badge';
+export {Badge, Pip} from './components/Badge';
```

Comment:
Good point.

The reason behind exposing `Pip` is that `Online Store` needs to indicate status in small layouts. It's not a strong use case, but we'd like to have it polaris due to the following:
- code already exists inside `Badge`
- easier to maintain UI changes when polaris updates

However `Badge` and `Pip` are somehow tethered to each other, making `Pip` more sub-component of `Badge`. 

So we wouldn't like promote `Pip` as a standalone "component" or highlight it as a pattern to be used here and there. 

It could be more a `Badge.Pip` sub-component, available to those specific consumers that need it without `Badge`